tests/integration/features/bootstrap/FeatureContext.php

```diff
@@ -577,7 +577,7 @@ public function userAcceptsDeclinesRemoteInvite(string $user, string $acceptsDec
 		if ($formData) {
 			if ($status === 200) {
 				if (!isset(self::$tokenToIdentifier[$response['token']])) {
-					self::$identifierToToken[$server . '::' . $roomName] = $response['token'];
```

Comment:
We need both?

Reply:
[`$identifierToToken` is already set in `userHasInvites`](https://github.com/nextcloud/spreed/blob/4e226349539c0b668a4bd60a3d994f8e9548aef1/tests/integration/features/bootstrap/FeatureContext.php#L552), which must be always called before calling accept/decline because [it also sets `$remoteToInviteId`](https://github.com/nextcloud/spreed/blob/4e226349539c0b668a4bd60a3d994f8e9548aef1/tests/integration/features/bootstrap/FeatureContext.php#L550)